data/856/711/91/85671191.geojson

```diff
@@ -818,7 +818,7 @@
     "wof:lang_x_spoken":[
         "fra"
     ],
-    "wof:lastmodified":1695880878,
+    "wof:lastmodified":1695885260,
```

Comment:
I see the lastmodified date is updated, but no changes for some of these features

Reply:
This looks like a 1% problem I'll manually reset.

More details in https://github.com/whosonfirst-data/whosonfirst-data-admin-gb/pull/90#discussion_r1341541471.

Reply:
We discussed in Slack and OK to skip as it's a 0.1% problem across all the related changes on 169k records that doesn't have negative consequences.